style.css

```diff
@@ -360,3 +348,107 @@ img {
 #get-intouch:active {
   color: green;
 }
+
+@media (min-width: 600px) {
+  .headline {
```

Comment:
- [ ] Kindly change this media query as said in the requirement, your current app is not responsive, take a look.

<img width="959" alt="ok" src="https://user-images.githubusercontent.com/77843050/206581918-8ff45ac6-e6d8-43cb-a928-5348f11a6c21.png">

![5](https://user-images.githubusercontent.com/77843050/206584023-78513c96-8eb9-4d50-9f7b-268893cce3e9.gif)